plugins/statistics.koplugin/main.lua

```diff
@@ -177,14 +178,25 @@ function ReaderStatistics:addToMainMenu(tab_item_table)
                 end
             },
             {
+                text = _("Reading progress"),
+                callback = function()
+                    UIManager:setDirty(nil, "full")
```

Comment:
The reason it works in emulator is because we always do full screen refresh on LCD since it's a very cheap operation. While with eink screens, partial refresh is preferred for faster screen refresh. It would nice if we can make the emulator also does partial screen refresh though, that will help uncover many screen refresh issues that will happen on the actual device.